README.md

```diff
@@ -1 +1,131 @@
-https://github.com/serhatbsts/ComputerScienceStudentsSummerHomeWork
+# Computer Science Students Summer Home-Work
+
+If you are a Computer Science student and want to improve your programming skills, you can start this project. This project will help you to improve your programming skills. You can do this project in your summer holiday.
```

Comment:
please update Read me information only related account manager project.
This all info is not related in current project

Reply:
OK, necessary corrections will be made.